### PR TITLE
Fix and simplify CUAV v5 nano debug description

### PR DESCRIPTION
**Pull request by hamishwillee**
This tidies/simplifies the CUAV v5 nano debug information. 
- It shows exactly what debug cable is provided.
- It explains in-place about the Vref not being in the cable.
- It links to System Console docs.
- @davids5 This also looks like a GH not SH type connector, so I changed that too.  Right?

FYI as discussed, I'm updating SWD info in autopilot docs first. I will have to come back and update this with link when I have created SWD debug setup doc.

### CODE REVIEW

File: en/flight_controller/cuav_v5_nano.md [hunk 1]
```diff
@@ -100,7 +99,17 @@ Pin | Signal | Volt
 5 (blk) | FMU_SWCLK | +3.3V
 6 (blk) | GND | GND
 
-> **Warning**  See also: [Using JTAG for hardware debugging](#issue_jtag)
+The product package includes a convenient debug cable that can be connected to the `DSU7` port.
+This spits out an FTDI cable for connecting the [PX4 System Console](https://dev.px4.io/master/en/debug/system_console.html) to a computer USB port, and SWD pins used for SWD/JTAG debugging.
```
Comment (davids5) [inline en/flight_controller/cuav_v5_nano.md]:
```suggestion
This splits out an FTDI cable for connecting the [PX4 System Console](https://dev.px4.io/master/en/debug/system_console.html) to a computer USB port, and SWD pins used for SWD/JTAG debugging.
```

File: en/flight_controller/cuav_v5_nano.md [hunk 2]
```diff
@@ -100,7 +99,17 @@ Pin | Signal | Volt
...
+
+![CUAV Debug cable](../../assets/flight_controller/cuav_v5_nano/cuav_nano_debug_cable.jpg)
+
+> **Warning** The SWD cable *does not split* out the 5V cable from the port.
```
Comment (davids5) [inline en/flight_controller/cuav_v5_nano.md]:
```suggestion
> **Warning** The SWD cable uses 5V as Vref but the CPU is run at 3,3V! 

  Some JTAG adapters (SEGGER J-Link) will use the Vref voltage to set the voltage on the SWD lines.   
```

File: en/flight_controller/cuav_v5_nano.md [hunk 3]
```diff
@@ -100,7 +99,17 @@ Pin | Signal | Volt
...
+>  For direct connection to *Segger Jlink* we recommended you connect the 3.3 Volts of `DSM`/`SBUS`/`RSSI` pin 4 as Pin 1 on the debug connector (`Vtref`).
```
Comment (davids5) [inline en/flight_controller/cuav_v5_nano.md]:
```suggestion
>  For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts from pin 4 of the connector marked `DSM`/`SBUS`/`RSSI` to connect to `Vtref` (Pin 1) of the debug cable. Thus providing 3.3V and *NOT* 5V to the JTAG.
```

**Comment by davids5** [comment]
> @davids5 This also looks like a GH not SH type connector, so I changed that too. Right?

Yep - it is a green GH

**Comment by hamishwillee** [comment]
Thanks very much for the review @davids5. I have integrated and very slightly modified your suggestion: https://github.com/PX4/px4_user_guide/pull/647/commits/5545c8dc7e628c6ab6a8ad5521a3c2192d2b814c

This clarifies that the provided cable does not even connect Vref and adjusts the other wording to match that understanding (previously this was kind of centered on the idea you were creating your own debug cable).

I think this is much better than it was, so now merging. I'll adjust CUAV v5 similarly when I get to it.

**Pull request merged**